third_party/render_globals.go

```diff
@@ -0,0 +1,18 @@
+package devrel_tutorial
+
+import (
+	"bytes"
+	"log"
+	"text/template"
+)
+
+// Reviewing comment: Templates can live anywhere within the repo
+const (
+  mdTmplsDir = 	"github.com/googlecodelabs/tools/third_party/templates/md/*"
+  htmlTmplsDir = 	"github.com/googlecodelabs/tools/third_party/templates/html/*"
```

Comment:
I don't know Go style but the spacing in these variable declarations seems off.

Reply:
Will rerun `gofmt`. I've see this constant-line-of-sight-to-value declarations in other files, but perhaps it's only for declaring `interfaces`